src/cli/record.ts

```diff
@@ -13,6 +14,8 @@ const builder = (args: yargs.Argv) =>
     .command(exportCommand)
     .command(importCommand)
     .command(deleteCommand)
+    .options(logOptions)
+    .middleware(logHandler)
```

Comment:
Because this logger must applied to the whole of this application, it's better to specify on `src/cli/main.ts`.